commands/command_fetch.go

```diff
@@ -278,6 +278,9 @@ func fetchAndReportToChan(pointers []*lfs.WrappedPointer, include, exclude []str
 		// This avoids previous case of over-reporting a requirement for files we already have
 		// which would only be skipped by PointerSmudgeObject later
 		passFilter := lfs.FilenamePassesIncludeExcludeFilter(p.Name, include, exclude)
+
+		lfs.IfNoLocalObjExistsLinkOrCopyFromReferenceMedia(p.Oid, p.Size)
```

Comment:
Is this early call solely for the `smudgeInfo`? I mean it will be called anyway later on by `PointerSmudge`.